ctx.go

```diff
@@ -450,7 +450,38 @@ func (c *DefaultCtx) Cookie(cookie *Cookie) {
 // The returned value is only valid within the handler. Do not store any references.
 // Make copies or use the Immutable setting to use the value outside the Handler.
 func (c *DefaultCtx) Cookies(key string, defaultValue ...string) string {
-	return defaultString(c.app.getString(c.fasthttp.Request.Header.Cookie(key)), defaultValue)
+	cookieValue := defaultString(c.app.getString(c.fasthttp.Request.Header.Cookie(key)), defaultValue)
+	// Parse the cookie value
+	value, ok := parseCookieValue(cookieValue)
+	if !ok {
+		return ""
+	}
+	return value
+}
+
+// parseCookieValue parses a cookie value according to RFC 6265.
+func parseCookieValue(raw string) (string, bool) {
+	// Strip the quotes, if present.
+	if len(raw) > 1 && raw[0] == '"' && raw[len(raw)-1] == '"' {
+		raw = raw[1 : len(raw)-1]
```

Comment:
The tests are not covering the case of having quotes which is the purpose of the PR.